airbyte-webapp/src/components/base/Switch/Switch.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import styles from "./Switch.module.scss";
+
+interface SwitchProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  small?: boolean;
+  loading?: boolean;
+}
+
+export const Switch: React.FC<SwitchProps> = ({ loading, small, checked, value, ...props }) => {
+  const isChecked = checked || !!value;
+  const labelStyle = `${styles.switch} ${small ? styles.small : ""} ${loading ? styles.loading : ""}`;
+  const spanStyle = `${styles.slider} ${small ? styles.small : ""} ${loading ? styles.loading : ""} ${
+    isChecked ? styles.checked : styles.unchecked
+  }`;
```

Comment:
Would this be cleaner with [classnames](https://github.com/JedWatson/classnames)?